dev/com.ibm.ws.kernel.feature.core/resources/com/ibm/ws/kernel/feature/internal/resources/ProvisionerMessages.nlsprops

```diff
@@ -226,6 +226,11 @@ UPDATE_DISABLED_FEATURES_ON_CONFLICT=CWWKF0046W: The configuration includes an i
 UPDATE_DISABLED_FEATURES_ON_CONFLICT.explanation=The configuration includes an incompatible combination of features. To avoid invalid behavior, the feature manager disabled all features.
 UPDATE_DISABLED_FEATURES_ON_CONFLICT.useraction=Specify a different version of the configured features that caused the conflict.
 
+# CWWK0044E  {0} = EeProgModel1: {1} = EeProgModel2: {2} = EeConflictFeature1: {3} = EeConflictFeature2: {4} = configuredFeature1: {5} configuredFeature2 {6} EePlaformCommonToProgModels
+UPDATE_INDIRECT_CONFLICT_INCOMPATIBLE_FEATURES_SAME_PLATFORM_ERROR=CWWKF0047E: The features {2} and {3} cannot be loaded at the same time.  The feature {2} of {0} is incompatible with the feature {3} of {1}. The configuration includes an incompatible combination of {6} features that is not supported.  The configured features {4} and {5} include an incompatible combination of features.  Your configuration is not supported.  Update the configuration to use compatible versions of the configured features that support the {0} or {1} programming models.
+UPDATE_INDIRECT_CONFLICT_INCOMPATIBLE_FEATURES_SAME_PLATFORM_ERROR.explanation=The server configuration contains an incompatible combination of features for the indicated programming models.  This conflict can happen if the two incompatible features are configured directly in the server configuration, or if the configured features include different versions of a feature that are incompatible.
```

Comment:
Remove the extra space after "indicated programming models".

Reply:
> Remove the extra space after "indicated programming models".

Done